views/layout.pug

```diff
@@ -32,4 +32,5 @@ html(lang="en")
       block content
     block scripts
       script(src="/jquery.min.js")
-      script(src="/js/bootstrap.min.js")
\ No newline at end of file
+      script(src="/js/bootstrap.min.js")
+      script(src="/clipboard-polyfill.js")
```

Comment:
Hmm... must this script be loaded on all pages?
Or just [append block](https://pugjs.org/language/inheritance.html) `scripts` in `dashboard.pug`?

Reply:
No it's not. Working on that.